trie/sync_bloom.go

```diff
@@ -193,22 +179,14 @@ func (b *SyncBloom) Contains(hash []byte) bool {
 		return true
 	}
 	// Bloom initialized, check the real one and report any successful misses
-	maybe := b.bloom.Contains(syncBloomHasher(hash))
+	maybe := b.bloom.ContainsHash(hashToUint64(hash))
 	if !maybe {
 		bloomMissMeter.Mark(1)
 	}
 	return maybe
 }
 
-// errorRate calculates the probability of a random containment test returning a
-// false positive.
-//
-// We're calculating it ourselves because the bloom library we used missed a
-// parentheses in the formula and calculates it wrong. And it's discontinued...
-func (b *SyncBloom) errorRate() float64 {
-	k := float64(b.bloom.K())
-	n := float64(b.bloom.N())
-	m := float64(b.bloom.M())
-
-	return math.Pow(1.0-math.Exp((-k)*(n+0.5)/(m-1)), k)
+func hashToUint64(h common.Hash) uint64 {
```

Comment:
It mixes all the bits in a single uint64, is it ok?

Reply:
That's the intent, yes. However, it's really not changed from before. The previous code took a `[]byte` as input, and called 
```golang
func (f syncBloomHasher) Sum64() uint64                     { return binary.BigEndian.Uint64(f) }
```
And this method here is exactly that same `binary.BigEndian.Uint64` method, but adapted for a fixed-size array input.